conf/xml/RobotStateProvider_iCub3_iFeel_SenseGlove.xml

```diff
@@ -0,0 +1,130 @@
+                                 "r_elbow",
+                                 "r_wrist_prosup",
+                                 "r_wrist_pitch",
+                                 "r_wrist_yaw",
+                                 "j_l_upper_arm_fake",
+                                 "j_r_upper_arm_fake",
+                                 "j_l_forearm_fake",
+                                 "j_r_forearm_fake",
+                                 "j_l_hand_fake",
+                                 "j_r_hand_fake",
+                                  "torso_pitch",
+                                 "torso_roll",
+                                 "torso_yaw")
+        </param>
+        <group name="CUSTOM_CONSTRAINTS">
+        <!-- check issue https://github.com/robotology/human-dynamics-estimation/issues/132 for more info-->
+        <!-- note that a group can not be empty, otherwise it returns error-->
+        <!-- custom joint limits velocities-->
+            <!--param name="custom_joints_velocity_limits_names">(neck_roll, neck_pitch)</param-->
+            <param name="custom_joints_velocity_limits_names">( )</param>
+            <!-- the upper bound is "+", while the lower bounds are "-" -->
+            <!--param name="custom_joints_velocity_limits_values">()</param-->
+            <param name="custom_joints_velocity_limits_values">( )</param>
+        <!-- **** base velocity limit: roll, pitch, yaw, x, y, z ****-->
+            <!--param name="base_velocity_limit_upper_buond">()</param>
+            <param name="base_velocity_limit_lower_buond">()</param-->
+        <!-- Custom joint Configuration constraints-->
+        <!-- if the boudary value is inf, I will use -1000.0 rad, or +1000.0 rad-->
+            <!--param name="custom_constraint_variables">()</param>
+            <param name="custom_constraint_matrix"> ()</param>
+            <param name="custom_constraint_lower_bound"> ()</param>
+            <param name="custom_constraint_upper_bound"> ()</param>
+            <param name="k_u">0.5</param>
+            <param name="k_l">0.5</param-->
+        </group>
+        <action phase="startup" level="5" type="attach">
+            <paramlist name="networks">
+                <elem name="HumanStateProviderLabel">XSenseIWearRemapper</elem>
+            </paramlist>
+        </action>
+        <action phase="shutdown" level="5" type="detach"/>
+    </device>
+
+    <!-- Uncomment to stream the output of HumanStateProvider on a YARP port -->
+    <device type="human_state_wrapper" name="RobotStateWrapper">
+        <param name="period">0.01</param>
+        <param name="outputPort">/iCub/RobotStateWrapper/state:o</param>
+        <action phase="startup" level="5" type="attach">
+            <paramlist name="networks">
+                <elem name="HumanStateWrapperLabel">RobotStateProvider</elem>
+            </paramlist>
+        </action>
+        <action phase="shutdown" level="5" type="detach"/>
+    </device>
+
+</robot>
```

Comment:
Until now, we didn't add any iFeel related configuration file in `human-dynamics-estimation`. I think we could either add all of them here, or don't add any for the time being and move this file with the other

Reply:
OK, I will add a PR to iFeel for this purpose.

Reply:
@lrapetti  I have added the other PR for this file.